src/connector/src/sink/remote.rs

```diff
@@ -174,11 +188,12 @@ async fn validate_remote_sink(param: &SinkParam) -> Result<()> {
                     | DataType::Jsonb
                     | DataType::Bytea
                     | DataType::List(_)
+                    | DataType::Struct(_)
```

Comment:
The `Struct` is only valid for es sink. We should reject struct type for other sinks.

Reply:
`array` only support `int16`, `int32`, `int64`, `float`, `double`, `varchar` type. How about reject other types as well?
https://github.com/risingwavelabs/risingwave/issues/13866